ui/src/warehouse.css

```diff
@@ -104,6 +104,29 @@ color: white;
 /*------------------------------------------------------------*/
 /* update bootstrap default tab */
 
+.brainlife-tab-dark .nav-tabs {
```

Comment:
I think it's better to organize this so that we can set different color by adding theme specific CSS by doing following

```
class="brainlife-tab brainlife-tab-dark"
```

This pattern is used by various other CSS framework, and it reduces duplicate CSS styling and makes it easier to update the style / add new theme, etc.